docs/plugins/index.md

```diff
@@ -82,10 +82,10 @@ Plugins may package static files to be served directly by the HTTP front end. En
 
 ### Restart WSGI Service
 
-Restart the WSGI service to load the new plugin:
+Restart the WSGI service to load the new plugin and restart rqworkers for using plugin in custom scripts:
```

Comment:
Please be sure to use proper grammar and punctuation when suggesting changes to the documentation.
```suggestion
Restart the WSGI service and RQ workers to load the new plugin:
```